youtube_dl/extractor/stopgame.py

```diff
@@ -0,0 +1,103 @@
+        'md5': 'a4d89bd7dc8f5db500b00894444e1bea',
+        'info_dict': {
+            'id': '390',
+            'ext': 'mp4',
+            'title': u'\u041f\u043e\u0441\u043b\u0435 \u0442\u0438\u0442\u0440\u043e\u0432. \xab\u0411\u0440\u0438\u043e\u043b\u0438\u043d\xbb',
+            'description': u'\u0412\u043e\u0442 \u0438 \u0434\u043e\u0431\u0440\u0430\u043b\u0438\u0441\u044c \u043d\u0430\u0448\u0438 \u0433\u043e\u043b\u043e\u0441\u0430 \u0434\u043e \u0442\u0440\u0443\u0434\u043e\u0432 \u0433\u0440\u0443\u043f\u043f\u0438\u0440\u043e\u0432\u043a\u0438 \u043c\u0443\u043b\u044c\u0442\u0438\u043f\u043b\u0438\u043a\u0430\u0442\u043e\u0440\u043e\u0432, \u043f\u0440\u043e\u0441\u043b\u0430\u0432\u0438\u0432\u0448\u0435\u0439\u0441\u044f \u0431\u043b\u0430\u0433\u043e\u0434\u0430\u0440\u044f \u0441\u0435\u0440\u0438\u0430\u043b\u0443 How It Should Have Ended. \u0412 \xab\u041f\u043e\u0441\u043b\u0435 \u0442\u0438\u0442\u0440\u043e\u0432\xbb \u0441\u043c\u044b\u0441\u043b \u043d\u0435\u0441\u043a\u043e\u043b\u044c\u043a\u043e \u0438\u043d\u043e\u0439 (\u043e \u0447\u0451\u043c \u0441\u0430\u043c\u043e \u043d\u0430\u0437\u0432\u0430\u043d\u0438\u0435 \u0438 \u0441\u043e\u043e\u0431\u0449\u0430\u0435\u0442), \u043d\u043e \u043c\u0435\u043d\u0435\u0435 \u0437\u0430\u0431\u0430\u0432\u043d\u044b\u043c\u0438 \u0432\u0438\u0434\u0435\u043e\u0440\u043e\u043b\u0438\u043a\u0438 \u0438\u0437-\u0437\u0430 \u044d\u0442\u043e\u0433\u043e \u043d\u0435 \u0441\u0442\u0430\u043d\u043e\u0432\u044f\u0442\u0441\u044f.',
+            'thumbnail': r'https://images.stopgame.ru/video/logos/84140.jpg'
+        }
+    }, {
+        'url': 'https://stopgame.ru/show/84140/posle_titrov_briolin',
+        'file': 'После титров. Бриолин-84140.mp4',
+        'md5': 'a4d89bd7dc8f5db500b00894444e1bea',
+        'info_dict': {
+            'id': '84140',
+            'ext': 'mp4',
+            'title': u'\u041f\u043e\u0441\u043b\u0435 \u0442\u0438\u0442\u0440\u043e\u0432. \xab\u0411\u0440\u0438\u043e\u043b\u0438\u043d\xbb',
+            'description': None,
+            'thumbnail': None
+        }
+    }, {
+        'url': 'https://stopgame.ru/videoplayer/84250',
+        'file': 'gamescom\ 2016.\ Геймплей-84250.mp4',
+        'md5': '1cc53dea8a6ac86eea9aa7a5d235a2f2',
+        'info_dict': {
+            'id': '84250',
+            'ext': 'mp4',
+            'title': 'gamescom 2016. Геймплей',
+            'description': None,
+            'thumbnail': None
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        try:
+            title = self._html_search_regex(r'<h1>(.+?)</h1>', webpage, 'title')
+            description = self._html_search_regex(r'<div itemprop="description">(.+?)</div>', webpage, 'description')
+            thumbnail = self._html_search_regex(r'<meta itemprop="thumbnail" content="(.+?)">', webpage, 'thumbnail')
+        except:
```

Comment:
Use fatal=False or default=None in _html_search_regex to handle fallbacks instead of try-except patterns.